### ISSUE TITLE
Please add MIPSEL architecture

### ISSUE THREAD

**Issue by iHardRock** [AppImage/AppImageKit#1166]
I think we need add support for mipsel arch. Standard compiler may be installed using 
**apt-get install gcc-mipsel-linux-gnu**

**Comment by TheAssassin** [comment]
- Have you tried to build AppImageKit for MIPS yet?
- What's the use case for MIPS? I am not aware of MIPS being in widespread use outside special domains like embedded systems, e.g., routers.
- Just installing a compiler is not enough. Cross-compiling can be very annoying. It'd make a lot more sense to build in a QEMU environment.

**Comment by iHardRock** [comment]
1. No, I have not try to build AppImageKit at all :smile: 
2. MIPS is a CPU architecture like ARM or x86, it may little bit less popular than ARM because there is no popular boards on MIPS like Raspberry Pi. But there are many small devices running Linux on MIPS.
3. I think its enough. Cross-compiling for MISP absolutly similar to ARM.

Long story short: 
I have use AppImage for a while to ship dependencies with my application for purpose of running on any Linux. For development I use Ubuntu 20.04, so my application compiles for this version, and then it able to run on Ubuntu 14.04 in production for example. Without any chnages and recompilation.
Today I have cross-compile my application for MIPS and i have found that AppImage have no support for MIPS.

Quic dive into source code gave me an answer why:
```
enum fARCH { 
    fARCH_i386,
    fARCH_x86_64,
    fARCH_arm,
    fARCH_aarch64
};
```

Its just not supported :)

**Comment by ShalokShalom** [comment]
Did you solve this?

**Comment by iHardRock** [comment]
No, I'm actually decided not to use AppImage on MIPS platforms because it not really necessarily. AppImage is simplification.

**Comment by probonopd** [comment]
Is there any current desktop machine that is still using MIPS?

**Comment by iHardRock** [comment]
AppImage may be used not only on desktops, but on servers too. This is a good way to deploy software. For example, my program, compiled under Ubuntu 20.04 and packed into AppImage (x86_64) is running on production Ubuntu 14.04 without any issues.
